server/utils/format.ts

```diff
@@ -38,4 +53,13 @@ export default {
       timeZone: 'Europe/London',
     })
   },
+
+  /**
+   * Formats dates in ISO style, used when calling APIs
+   *
+   * Example: `2024-07-30`
+   */
+  isoDate(date: Date): string {
```

Comment:
`isoDate()` is a better name :)

I think this is fine, but I guess the `&&` may silently "fail" and return something invalid, e.g. `false && ` would return `false` rather than an empty string 🤔 (before it would throw an exception)

We could leave as is :) - or add a check for at least `undefined`/`null` - but I don't think we have to be super-rigorous on this one, JS is a bit of a mess when it comes to type checking (anything is an `object`)